### PR TITLE
Update CEDS 

### PR DESCRIPTION
**Pull request by znichollscr**
## Checklist

Please confirm that this pull request has done the following:

- [x] Documentation added (where applicable)
- [x] Changelog item added to `changelog/`

### CODE REVIEW

**Review by durack1 — APPROVED** [review]
Great, looks good to me! If we can resync this after 6 pm PST/US west coast tonight, then I think we'll be good to go. All retractions and new publications are now done

**Comment by znichollscr** [comment]
Good to merge @durack1

**Comment by znichollscr** [comment]
Up to you whether you want to wait for any potential feedback on the docs from the CEDS team or not (we can always update in future)

File: docs/dataset-overviews/anthropogenic-slcf-co2-emissions.md
Comment (durack1) [inline docs/dataset-overviews/anthropogenic-slcf-co2-emissions.md:1]: @hahsan1 @ssmithClimate just checking this with you guys - if there are no tweaks to make, we'll merge this tomorrow, which will show up on the input4MIPs website - https://input4mips-controlled-vocabularies-cvs.readthedocs.io/en/latest/dataset-overviews/anthropogenic-slcf-co2-emissions/

**Pull request merged**